stacks/st2.yaml

```diff
@@ -24,7 +26,8 @@ st2express:
     - 172.168.50.11
   mounts:
     - '/opt/stackstorm:artifacts/stackstorm'
-  vm_sync: nfs
+    - '/opt/ansible:artifacts/ansible'
```

Comment:
It's not that. It's that by default a stack shouldn't mount or assume NFS mounts. These are user configurable options, and you can/should document it in `template.yaml` for others to use. Also, documents on how to use this are always a plus, but I won't hold up the PR for documents.

And as far as moving the Puppet code. Sure. It is possible to do this. Feel free to submit a PR, or I'll take a pass at it next time I sit down with this code.